docs/testing.md

```diff
@@ -86,3 +86,12 @@ for oCIS acceptance tests.
 #### Watch the test run
 
 To watch the tests while running, open [http://host.docker.internal:6080/](http://host.docker.internal:6080/) in the browser to access your VNC client.
+
+
+#### Watch the test report
+
+To generate test report, should runing test with:  ``` -f json:tests/report/cucumber_report.json && node tests/report/index.js ```
+
+command ``` -f json:tests/report/cucumber_report.json ``` creates or overwrites the file cucumber_report.json in the folder "tests/report"
```

Comment:
i would add the `-f json:tests/report/cucumber_report.json` option as a default one to `test:acceptance:oc10` and `test:acceptance:ocis`. Then the report json is always in place. 

Then we could rewrite the docs to:
... after a successful testrun you can inspect the report by running `node tests/report/index.js`

please do not forget to add `cucumber_report.json` and `cucumber_report.html` to the `.gitignore`